frontend/e2e-tests/page-objects/data_entry/DataEntryHomePgObj.ts

```diff
@@ -2,7 +2,7 @@ import { expect, type Locator, type Page } from "@playwright/test";
 
 import { PollingStation } from "@/api";
 
-export class DataEntryChoicePage {
+export class DataEntryHomePage {
```

Comment:
The feature in the frontend is called 'data_entry_choice', do we also want to rename it there?